### DISCUSSION TITLE
GitHub links isn’t loading on my computer

### DISCUSSION THREAD

**Discussion by HelloHiGitHub** [community/community#39462] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![5BC2D7BB-2282-455C-A9D0-9C4579B20ABC](https://user-images.githubusercontent.com/118566760/202790864-5c53f75b-e98e-4802-b192-f5c62803bd27.jpeg)

Everytime I try to load a GitHub link on my computer it says that (the picture linked to this) I tried different browsers it says the same thing, any other website works EXCEPT github

**Comment by SilasRodrigues19** [comment]
@HelloHiGitHub Try to clear the DNS cache, with the following commands in the terminal

`ipconfig /flushdns`

`ipconfig /registerdns`

`ipconfig /release`

`ipconfig /renew`

`netsh winsock reset`

**Reply by HelloHiGitHub** [reply]
Didn’t work, thanks though.
